lib/streamlit/web/server/upload_file_request_handler.py

```diff
@@ -19,21 +19,79 @@
 
 from streamlit import config
 from streamlit.logger import get_logger
+from streamlit.runtime.memory_uploaded_file_manager import MemoryUploadedFileManager
 from streamlit.runtime.uploaded_file_manager import UploadedFileManager, UploadedFileRec
 from streamlit.web.server import routes, server_util
 
-# /_stcore/upload_file/(optional session id)/(optional widget id)
-UPLOAD_FILE_ROUTE = (
-    r"/_stcore/upload_file/?(?P<session_id>[^/]*)?/?(?P<widget_id>[^/]*)?"
-)
+UPLOAD_FILE_URLS_ROUTE = r"/_stcore/upload_urls/"
+
 LOGGER = get_logger(__name__)
 
 
+class UploadFileUrlsRequestHandler(tornado.web.RequestHandler):
```

Comment:
would be good to have a comment here saying that we may not need this (+ it's looking like we'll be going down the websocket route so I'm guessing this can be removed soon-ish)